hydroflow/tests/surface_codegen.rs

```diff
@@ -114,6 +115,77 @@ pub fn test_large_diamond() {
     df.run_available();
 }
 
+#[test]
+pub fn test_split() {
+    use hydroflow::{tl, tt};
+
+    let (out_send_a, mut out_recv_a) = hydroflow::util::unbounded_channel::<usize>();
+    let (out_send_b, mut out_recv_b) = hydroflow::util::unbounded_channel::<usize>();
+
+    fn split_even_odd(x: usize) -> tt!(Option<usize>, Option<usize>) {
+        if 0 == x % 2 {
+            tl!(Some(x), None)
+        } else {
+            tl!(None, Some(x))
+        }
+    }
```

Comment:
`split()` operator allows sending to both outputs simultaneously, use `Option` with `-> flatten()` to send to specific outputs